.codecov.yml

```diff
@@ -1,17 +1,14 @@
 # Documentation: https://docs.codecov.io/docs/codecov-yaml
 
-codecov:
-  bot: dlang-bot
-
 coverage:
   precision: 3
-  # round: down
-  # range: "70...100"
+  round: down
+  range: "80...100"
 
   status:
     # Learn more at https://docs.codecov.io/docs/commit-status
     project: true
     patch: true
-    changes: false
+    changes: true
```

Comment:
Not sure about the effects/usefulness of this one, but I guess it doesn't hurt to try.